cmake/modules/FindLEX.cmake

```diff
@@ -22,6 +22,11 @@
 find_program(LEX_EXECUTABLE NAMES lex DOC "Path to the lex executable")
 mark_as_advanced(LEX_EXECUTABLE)
 
+if(LEX_EXECUTABLE)
+else()
+	find_program(LEX_EXECUTABLE NAMES flex DOC "Path to the lex executable (as flex)")
+endif()
```

Comment:
This may not the right place to do this. This is a modular package for `lex` specifically, which is not same as `flex`. We may want to consider instead in the Tril CMakeLists.txt [1] not setting the `REQUIRED` tag for `lex` and `yacc` and as a fallback using `flex` and `bison` similarly to how we used to do it in 0c2f0d03d5bb63859acd1426e71829622a91c31b. It's going to be slightly more code, but a lot cleaner solution without any surprises.

[1] https://github.com/eclipse/omr/blob/c93c18e30cda68e7c5b592b6150c133ce8c6aa93/fvtest/tril/tril/CMakeLists.txt#L29-L30

Reply:
How did the Windows tests not fail in the PR which removed Flex and Bison?

Reply:
I question that myself. Perhaps it's a Cygwin vs. native thing? I'll let @wbh123456 or @Leonardo2718 answer though.

Reply:
Tril isn't being built on Windows.

Reply:
Ah, indeed that's why. We do build JitBuilder, but not Tril. That explains it. We are using the AppVeyor [1] [2] specification while the others use the Travis one. AppVeyor does not build Tril and is less complete. 

On a side note are we investing into moving Windows towards building all targets? i.e. closing the gap between the Travis and AppVeyor caches?

[1] https://github.com/eclipse/omr/blob/de8e7b0baf031c127e2f2a6f6e7d7343ab73b069/buildenv/jenkins/jobs/pull-requests/PullRequest-win_x86-64#L18-L24
[2] https://github.com/eclipse/omr/blob/de8e7b0baf031c127e2f2a6f6e7d7343ab73b069/cmake/caches/Appveyor.cmake

Reply:
> On a side note are we investing into moving Windows towards building all targets? i.e. closing the gap between the Travis and AppVeyor caches?

Not as far as I'm aware of, although @wbh123456 is working on getting Tril to build and run tine AppVeyor builds.